update.go

```diff
@@ -303,3 +371,40 @@ other options are ignored.
 		return container.Set(config)
 	},
 }
+
+func stringSliceToThrottleDevice(ss []string) ([]specs.LinuxThrottleDevice, error) {
+	tds := make([]specs.LinuxThrottleDevice, 0, len(ss))
+	for _, v := range ss {
+		parts := strings.SplitN(strings.TrimSpace(v), " ", 2)
+		if len(parts) != 2 {
+			return nil, fmt.Errorf("format error: %s", v)
```

Comment:
please use the format:

```
fmt.Errorf("failed to parse throttle device %s: concrete reason", ss)
```

please do not missing the concrete reason. Since the added reason is the most important thing for users to understand. @zjumoon01

Reply:
what about the two error messages printed as below:
fmt.Errorf("failed to parse throttle device %s, the expected format is 'major:minor rate'", v)

Reply:
much better, now.